core/src/main/res/values/arrays.xml

```diff
@@ -234,9 +234,9 @@
     </string-array>
 
     <string-array name="media_player_options">
+        <item>@string/media_player_exoplayer_recommended</item>
         <item>@string/media_player_builtin</item>
         <item>@string/media_player_sonic</item>
-        <item>@string/media_player_exoplayer_recommended</item>
```

Comment:
`media_player_values` should be changed, too. Otherwise the actual settings do not fit what is displayed

Reply:
done